bird-watcher/bird-watcher.js

```diff
@@ -0,0 +1,54 @@
+// @ts-check
+//
+// The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion when
+// implementing this exercise.
+
+/**
+ * Calculates the total bird count.
+ *
+ * @param {number[]} birdsPerDay
+ * @returns {number} total bird count
+ */
+
+export function totalBirdCount(birdsPerDay) {
+  let sum = 0;
+  for(let i = 0; i < birdsPerDay.length; i++) {
+    sum += birdsPerDay[i];    
+  }
+  return sum;
+}
+
+/**
+ * Calculates the total number of birds seen in a specific week.
+ *
+ * @param {number[]} birdsPerDay
+ * @param {number} week
+ * @returns {number} birds counted in the given week
+ */
+export function birdsInWeek(birdsPerDay, week) {
+  let sum = 0;
+  let startIndex = 7 * (week - 1);
+  let endIndex = 7 * week;
+  for(let i = startIndex; i < endIndex; i++){
```

Comment:
Would you mind leaving an space between the closing parenthesis and the opening brackets?